.gitignore

```diff
@@ -1,4 +1,4 @@
-/target
+target/
```

Comment:
Personally I think it would be better to have just single top-level `.gitignore` with `target/` and list the possible exceptions there (e.g. if we need to include some `target` dir). But that's just my view.